script/hassfest/services.py

```diff
@@ -35,6 +36,9 @@ def exists(value):
 SERVICE_SCHEMA = vol.Schema(
     {
         vol.Required("description"): str,
+        vol.Optional("target"): vol.Any(
+            selector.TargetSelector.CONFIG_SCHEMA, None  # pylint: disable=no-member
+        ),
```

Comment:
This pylint disable should not be needed (tested locally)
```suggestion
        vol.Optional("target"): vol.Any(selector.TargetSelector.CONFIG_SCHEMA, None),
```

Reply:
nvm... `pylint` !== `mypy`. Good morning!